### PR TITLE
Docs: Docs for the Docs

### PR DESCRIPTION
**Pull request by stanleygjones**
**Overview**

To set up this repository for some contributors, I've added some baseline docs for contributing.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+**Overview**
```
Comment (stanleygjones) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: This template is based on @hansl 's design template, stripped down to pull request size.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
-# docs
-Public documentation
+# Write code for the Internet Computer
```
Comment (stanleygjones) [inline README.md:1]: Dual purpose of the README. First as the entry to the actual documentation, second as the landing page for people who want to contribute to it.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+- [*Get Samples*](https://sdk.dfinity.org/docs/developers-guide/sample-apps.html) Access sample code, applications, and microservices.
+- [*Developer Tools*](https://sdk.dfinity.org/docs/developers-guide/introduction-key-concepts.html) Take a deeper dive into how you can use the SDK to create applications.
+- [*Language Guide*](https://sdk.dfinity.org/docs/language-guide/motoko.html) Explore Motoko, the  programming language for the Internet Computer.
+- [*Basic Tutorials*](https://sdk.dfinity.org/docs/developers-guide/tutorials-intro.html) Learn the basics with simple examples and step-by-step instructions for first-timers.
+
+# Contributing to the documentation
+
+The Internet Computer is a new technology stack that is unhackable, fast, scales to billions of users around the world, and supports a new kind of autonomous software that promises to reverse Big Tech’s monopolization of the internet. It allows developers to take on the monopolization of the internet, and return the internet back to its free and open roots. We're committed to connecting those who believe the same through our events, content, and discussions.
```
Comment (stanleygjones) [inline README.md:14]: This language is taken straight from the website.

File: .github/CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
+
+# Contributor Covenant Code of Conduct
```
Comment (stanleygjones) [inline .github/CODE_OF_CONDUCT.md:2]: A very popular template for codes of conduct. Code of conducts?

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+Thank you for your interest in contributing to the Internet Computer. By participating in this project, you agree to abide by our [Code of Conduct](./CODE_OF_CONDUCT.md).
```
Comment (stanleygjones) [inline .github/CONTRIBUTING.md:3]: This doc needs some help. @lsgunnlsgunn did you say you had something in the works? I'd be happy to adapt it.

File: LICENSE [hunk 6]
```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2020 DFINITY Stiftung, ALL RIGHTS RESERVED
```
Comment (stanleygjones) [inline LICENSE:1]: This license had been written for a different repo. Repurposing it here.

File: README.md (hunk 3 above)
Comment (alexabsmith) [reply README.md:14]: To reduce some of the "monopolization," here's an alternative also from some website copy:

It allows developers to create open, autonomous software and services and restores the Internet to its permissionless, innovative, and collaborative roots.

File: LICENSE (hunk 6 above)
Comment (kritzcreek) [reply LICENSE:1]: Not a lawyer, but in my experience if you want to use this kind of a license _and_ accept contributions you need to make every contributor sign a CLA (Contributer License Agreement).

File: .github/CONTRIBUTING.md (hunk 5 above)
Comment (lsgunnlsgunn) [reply .github/CONTRIBUTING.md:3]: I had a rough draft here:
https://github.com/dfinity-lab/documentation/blob/LG/DRAFT-contributing/CONTRIBUTING.md

Comment (lsgunnlsgunn) [reply .github/CONTRIBUTING.md:3]: added a draft to this PR.

**Pull request merged**